.github/workflows/release.yml

```diff
@@ -0,0 +1,128 @@
+      - name: Check if Dapr Cli's version is new.
+        run: |
+          NEW_VERSION=$( cat dapr-cli.rb | grep "version " | grep "'${DAPR_CLI_VERSION}'" &> /dev/null || echo "yes" )
+          echo "##[set-env name=NEW_VERSION;]$NEW_VERSION"
+      - name: Get Dapr Cli's source code tarball's shasum.
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+        if: env.NEW_VERSION == 'yes'
+        run: |
+          DAPR_CLI_SRC_SHASUM=$( curl -L -f "https://x-access-token:${GITHUB_TOKEN}@github.com/dapr/cli/archive/v${DAPR_CLI_VERSION}.tar.gz" 2> /dev/null | shasum -a 256 | sed -n 's/\(.*\) -/\1/p' | xargs -I {} echo {} )
+          echo "##[set-env name=DAPR_CLI_SRC_SHASUM;]$DAPR_CLI_SRC_SHASUM"
+      - name: Get Homebrew's tap for Dapr Cli.
+        if: env.NEW_VERSION == 'yes'
+        run: brew tap dapr/tap && brew update
+      - name: Generating initial formula for new version.
+        if: env.NEW_VERSION == 'yes'
+        run: bash ./.github/scripts/generate-temp-formula.sh
+      - name: Generating Homebrew's bottle file for Dapr Cli.
+        if: env.NEW_VERSION == 'yes'
+        run: |
+          brew install dapr-cli --build-bottle && brew bottle dapr-cli
+          generated_bottle_filename=$( ls -1 *.bottle.*.tar.gz )
+          bottle_filename=$( echo $generated_bottle_filename | sed -e 's/--/-/' | sed -e 's/bottle.*.tar.gz/bottle.tar.gz/' )
+          DAPR_CLI_BOTTLE_OS_VERSION=$( echo $generated_bottle_filename | sed -e 's/--/-/' | sed -e 's/bottle.*.tar.gz/bottle.tar.gz/' | sed -e 's/.*\.\(.*\)\.bottle.*/\1/' )
+          mv $generated_bottle_filename $bottle_filename
+          DAPR_CLI_BOTTLE_SHASUM=$( shasum -a 256 ${bottle_filename} | sed -n "s/\(.*\) ${bottle_filename}/\1/p" | xargs -I {} echo {} )
+          mkdir -p release
+          mv $bottle_filename release/$bottle_filename
+          echo ${DAPR_CLI_VERSION} > release/version.txt
+          echo "##[set-env name=DAPR_CLI_BOTTLE_OS_VERSION;]$DAPR_CLI_BOTTLE_OS_VERSION"
+          echo "##[set-env name=DAPR_CLI_BOTTLE_SHASUM;]$DAPR_CLI_BOTTLE_SHASUM"
+      - name: Generating initial formula for new version.
+        if: env.NEW_VERSION == 'yes'
+        run: bash ./.github/scripts/generate-formula.sh
+      - name: Release artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: bottle_drop
+          path: ./release
+      - name: Commit to repository
+        if: env.NEW_VERSION == 'yes'
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+          COMMIT_MSG: |
+            Updating dapr-cli formula for version ${{ env.DAPR_CLI_VERSION }}
+            skip-checks: true
+        run: |
+          git config user.email "Dapr Bot"
+          git config user.name "daprweb@microsoft.com"
+          # Update origin with token
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
```

Comment:
Make sure that GITHUB_TOKEN is not exposed to the log.

Reply:
It is not:

```
2020-04-30T21:16:14.7921706Z   },
2020-04-30T21:16:14.7921930Z   request: {
2020-04-30T21:16:14.7922326Z     method: 'POST',
2020-04-30T21:16:14.7922881Z     url: 'https://api.github.com/repos/artursouza/homebrew-tap/releases',
2020-04-30T21:16:14.7923196Z     headers: {
2020-04-30T21:16:14.7924312Z       accept: 'application/vnd.github.v3+json',
2020-04-30T21:16:14.7924812Z       'user-agent': 'octokit.js/16.43.1 Node.js/12.16.1 (Linux 5.0; x64)',
2020-04-30T21:16:14.7927802Z       authorization: 'token [REDACTED]',
2020-04-30T21:16:14.7928483Z       'content-type': 'application/json; charset=utf-8'
2020-04-30T21:16:14.7928789Z     },
```